lib/podium.ex

```diff
@@ -72,11 +71,11 @@ defmodule Podium do
       |> remove_nils()
       |> inject_application_uid()
 
-    message = %{
+    params = %{
```

Comment:
Can we undo all the renaming so that we're only changing the things that need to be changed?